Sources/Host/Darwin/LibProc.cpp

```diff
@@ -55,7 +58,23 @@ void LibProc::EnumerateProcesses(
 }
 
 std::string LibProc::GetThreadName(ProcessId pid, ThreadId tid) {
-  return "<unknown>";
+  std::string dft("<unknown-thread>");
+  thread_identifier_info_data_t threadId;
+  struct proc_threadinfo ti;
+  ErrorCode err;
+  Mach mach(pid);
```

Comment:
Sucks a little bit that we have to create a whole new `Mach` object here just to get a thread name.